modules/AlphaWalletFoundation/AlphaWalletFoundation/EtherClient/TransactionSigning.swift

```diff
@@ -3,70 +3,210 @@
 import BigInt
 import CryptoSwift
 
-public protocol Signer {
-    func hash(transaction: UnsignedTransaction) throws -> Data
-    func values(signature: Data) -> (r: BigInt, s: BigInt, v: BigInt)
+public struct EthereumSignature {
+    public let v: UInt8
+    public let r: [UInt8]
```

Comment:
Do we need it to be `UInt8` and `[UInt8]`? Seems simpler if we keep them to `Data` and convert where needed?